.nyc_output/processinfo/index.json

```diff
@@ -0,0 +1 @@
+{"processes":{"db1a29ad-0bc4-46af-bd29-b5de119bbdb8":{"parent":null,"children":[]}},"files":{"C:\\Users\\Admin\\Desktop\\Politico\\server\\tests\\users-tests.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\app.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\routes\\mainRouter.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\routes\\usersRouter.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\controllers\\users.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\helpers\\validation.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\helpers\\messages-and-codes.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\helpers\\queries.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"],"C:\\Users\\Admin\\Desktop\\Politico\\server\\data\\config.js":["db1a29ad-0bc4-46af-bd29-b5de119bbdb8"]},"externalIds":{}}
```

Comment:
would you mind putting this folder (`.nyc_output`) in git ignore??

Reply:
I have fixed it!